folly/portability/Stdlib.cpp

```diff
@@ -102,12 +102,19 @@ int setenv(const char* name, const char* value, int overwrite) {
 
   // Here lies the documentation we blatently ignore to make
   // this work >_>...
-  *getenv(name) = '\0';
+
+  char* env = getenv(name);
+  if (!env) {
+    errno = EINVAL;
+    return -1;
+  }
+  *env = '\0';
+  *(env + 1) = '=';
```

Comment:
This needs to be back below this next comment on line 114.

Reply:
I'm also not entirely convinced the change here is required, since we already call `_putenv_s` for this environment variable up on line 98, so the variable should be guaranteed to exist.